c7n/resources/cfn.py

```diff
@@ -72,6 +72,41 @@ def process_stacks(self, stack):
         client.delete_stack(StackName=stack['StackName'])
 
 
+@CloudFormation.action_registry.register('disable-protection')
+class DisableProtection(BaseAction):
+    """Action to disable termination protection
```

Comment:
fwiw, generally we've trying to move away from disable/enable style towards a `set-protection` that can cover off on both use cases, it helps reduce the size of the vocabulary that users have to learn.